src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,444 @@
+
+### File formats
+The Microscopy community uses a variety of formats for storing raw data, and there is no single
+standard that all researchers agree on. However, a standardized file structure has been developed
+by the [Open Microscopy Environment](https://www.openmicroscopy.org/) for whole-slide imaging with
+the [OME-TIFF file specifications](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/file-structure.html).
+The OME-TIFF file allows for multi-page TIFF files to store multiple image planes and supports
+multi-resolution pyramidal tiled images. An OME-XML data block is also embedded inside the
+file’s header.
+
+The BIDS standard accepts microscopy data in different file formats to accommodate datasets
+stored in 2D image formats and whole-slide imaging formats, to accommodate lossless and lossy
+compression, and to avoid unnecessary conversions of the original data from a non-tiled to a
+tiled format, or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats, for example with the
+Next-Generation File Formats currently developed by OME ([OME-NGFF](https://ngff.openmicroscopy.org/latest/))
+as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+
+{{ MACROS___make_suffix_table(
+      [
+         "TEM",
+         "SEM",
+         "uCT",
+         "BF",
+         "DF",
+         "PC",
+         "DIC",
+         "FLUO",
+         "CONF",
+         "PLI",
+         "CARS",
+         "2PE",
+         "MPE",
+         "SR",
+         "NLO",
+         "OCT",
+         "SPIM",
+      ]
+   )
+}}
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
+Consistent with other data types in BIDS, the session entity is optional.
+
+The [`sample-<label>`](../99-appendices/09-entities.md#sample) entity is REQUIRED for
+Microscopy data and is used to distinguish between different samples from the same subject.
+The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+
```

Comment:
```suggestion
"Sample" concept may be similar or equivalent, to other terms, such as “tissue” or “cell”.
```
Same here, maybe is good to mention some similar terms that people may use, so they easily feel recognized?

Reply:
The link of `sample-<label>` goes to the definition of sample in the [appendix](https://bids-specification--881.org.readthedocs.build/en/881/99-appendices/09-entities.html#sample) which is "A sample pertaining to a subject such as tissue, primary cell or cell-free sample.". I would prefer not to duplicates definition in this case.